eos/rare-b-decays/lambda-b-to-lambda-nu-nu.cc

```diff
@@ -36,46 +40,33 @@ namespace eos
 
         std::shared_ptr<FormFactors<OneHalfPlusToOneHalfPlus>> form_factors;
 
-        Parameters parameters;
-
         UsedParameter m_Lambda_b;
-
         UsedParameter tau_Lambda_b;
-
         UsedParameter m_Lambda;
-
         UsedParameter g_fermi;
-
         UsedParameter alpha_e;
-
         UsedParameter hbar;
-
         UsedParameter mu;
 
+        using IntermediateResult = LambdaBToLambdaDineutrino::IntermediateResult;
+        IntermediateResult intermediate_result;
+
         static const std::vector<OptionSpecification> options;
 
         std::function<complex<double> ()> lambda_t;
         std::function<WilsonCoefficients<wc::SBNuNu> ()> wc;
 
-        GSL::QAGS::Config int_config;
-
         bool cp_conjugate;
 
         Implementation(const Parameters & p, const Options & o, ParameterUser & u) :
             model(Model::make(o.get("model", "SM"), p, o)),
-            parameters(p),
             m_Lambda_b(p["mass::Lambda_b"], u),
             tau_Lambda_b(p["life_time::Lambda_b"], u),
-<<<<<<< Updated upstream
```

Comment:
This needs to be edited in the previous commit, so that the code compiles for every individual commit. (think: ``git bisect``)